tools/lambda-promtail/lambda-promtail/s3.go

```diff
@@ -115,13 +115,16 @@ func getLabels(record events.S3EventRecord) (map[string]string, error) {
 
 func processS3Event(ctx context.Context, ev *events.S3Event) error {
 
+	fmt.Println("processing new S3Event\n")
```

Comment:
`Println()` already adds a newline
```suggestion
	fmt.Println("processing new S3Event")
```